README.md

```diff
@@ -1,17 +1,100 @@
 # Conda Recipes
 
-Example recipes for the new conda build system.  Use
+Community-maintained repository of conda package build recipes
+for 592+ conda packages.
 
-    conda build <recipe directory>
+* Package maintainers SHOULD upgrade packages
+  when appropriate.
 
-You have to use conda >= 1.7
-[conda](https://github.com/continuumio/conda).
+* Conda recipes MAY run other scripts:
 
-See http://docs.continuum.io/conda/build.html for information on how to make a recipe,
-or just look at the examples here.
 
-This project (`conda-recipes`) is in the public domain.   Note that this statement
+  * at build time by specifying ``script:`` in ``meta.yaml``
+    (default: ``build.bat`` / ``build.sh``)
+  * at build time in `build.bat` (Windows) and `build.sh` (OSX, Linux)
+  * before the package is installed in ``pre-link.sh``
+  * after the package is installed in ``post-link.sh``
+  * before the package is removed in ``pre-unlink.sh``
+
+  http://conda.pydata.org/docs/build.html
+
+* Some of these recipes do intersect with
+  [Conda packages included with Anaconda](http://docs.continuum.io/anaconda/pkg-docs.html).
+
+* You can [search for Conda packages hosted in free public BinStar Channels](https://binstar.org/search?q=).
+
+
+## Requirements
+
+* [conda](https://github.com/conda/conda) >= 1.7
+  -- https://github.com/conda/conda#installation
+
+* [conda-build](https://github.com/conda/conda-build)
+  -- https://github.com/conda/conda-build#installation
+
+## Creating Conda Recipes
+See http://conda.pydata.org/docs/build.html for information
+on how to make a conda recipe, or just look at the [examples](#examples) here.
+
+    # Generate a conda recipe from PyPI (Python)
+    cd conda-recipes/
+    conda skeleton pypi arrow
+    conda skeleton pypi rdflib-jsonld
+
+    # Generate a conda recipe from CRAN (R)
+    cd conda-recipes/r-packages
+    conda skeleton cran r-chron
+    conda skeleton cran r-dplyr
+
+    # Generate a conda recipe from CPAN (Perl)
+    conda skeleton cpan DateTime
+    conda skeleton cpan RDF::Helper
```

Comment:
I think you could explain a bit more what `skeleton` does in this section. I mean, you could say that users can automatically generate recipes for simple Python, R and Perl packages by using `skeleton`, and then give the examples below.